### PR TITLE
Update docs: initial page updates

### LINKED ISSUE #138: `EnsembleKalmanProcesses.jl` v0.5 dependency breaks for me
Fix: Update to v0.6 in compat.

### PR DESCRIPTION
**Pull request by odunbar**
## Purpose

Fixes #138 and adds some new docs to kickoff work in #136 
Had to update EKP to a newer version which lead to small adjustment to MCMC and GP testing
## In this PR
- [x] Clima format update
- [x] Adds Clima Logo, authorship notice
- [x] Updates the home-page

### CODE REVIEW

**Comment by odunbar** [comment]
@tsj5 confirmed changes offline... merging

**Comment by odunbar** [comment]
bors try

File: examples/Lorenz/Project.toml [hunk 1]
```diff
@@ -17,4 +17,4 @@ StatsPlots = "f3b207a7-027a-5e70-b257-86293d7955fd"
 
 [compat]
 FFTW = "= 1.3.0"
-julia = "~1.6"
+julia = "^1.6"
```
Comment (charleskawczynski) [inline examples/Lorenz/Project.toml:20]: cc @odunbar no need to change?

File: examples/Emulator/GaussianProcess/Project.toml [hunk 2]
```diff
@@ -1,5 +1,4 @@
 [deps]
-CalibrateEmulateSample = "95e48a1f-0bec-4818-9538-3db4340308e3"
```
Comment (charleskawczynski) [inline examples/Emulator/GaussianProcess/Project.toml:2]: cc @odunbar revert

File: Project.toml [hunk 3]
```diff
@@ -21,8 +21,8 @@ Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 StatsBase = "2913bbd2-ae8a-5f71-8c99-4fb6c76f3a91"
 
 [compat]
-EnsembleKalmanProcesses = "^0.5"
-julia = "~1.6"
+EnsembleKalmanProcesses = "^0.8"
+julia = "^1.6"
```
Comment (charleskawczynski) [inline Project.toml:25]: cc @odunbar revert

File: examples/Emulator/GaussianProcess/Project.toml (hunk 2 above)
Comment (odunbar) [reply examples/Emulator/GaussianProcess/Project.toml:2]: ok dev'ing the local package here

File: examples/Lorenz/Project.toml (hunk 1 above)
Comment (odunbar) [reply examples/Lorenz/Project.toml:20]: Reverted

File: Project.toml (hunk 3 above)
Comment (odunbar) [reply Project.toml]: Reverted Julia but not EKP - we wish to have 0.8 from now on anyway

**Comment by odunbar** [comment]
bors try

**Comment by odunbar** [comment]
approved offline

**Pull request merged**